docs/src/versions.md

```diff
@@ -4,11 +4,20 @@ This version includes many **breaking changes**.
 
 Added support for Dipole-mode Linear Spin Wave Theory. (Thanks Hao Zhang!)
 
-Split `intensities` into calculation ([`intensity_formula`](@ref)) and presentation ([`intensities_interpolated`](@ref), [`intensities_binned`](@ref)). This is a **breaking change**, see the docs to migrate your code.
+Split `intensities` into calculation ([`intensity_formula`](@ref)) and
+presentation ([`intensities_interpolated`](@ref), [`intensities_binned`](@ref)).
+This is a **breaking change**, see the docs to migrate your code.
 
-Broadened support for custom observables in `StructureFactor` for use in `intensity_formula`.
+`StructureFactor` type renamed `SampledCorrelations`. An appropriate
+`SampledCorrelations` is created by calling either `dynamical_correlations` or
+`instant_correlations` instead of `DynamicStructureFactor` or
```

Comment:
Add hyperlinks,
```
[`SampledCorrelations`](@ref)
```
and similarly for `dynamical_correlations` and `instant_correlations`?

Reply:
Thanks. Added. (Note that the documentation to `SampledCorrelations` is basically just a stub and points to the docstrings for `dynamical_intensities` and `instant_intensities`. But I think it is useful to export `SampledCorrelations`, just because so many locations in the docs really want to refer to `SampledCorrelations` itself.)